libs/sdk-ui-tests/stories/visual-regression/kit/Header/Header.tsx

```diff
@@ -4,7 +4,7 @@ import { storiesOf } from "@storybook/react";
 import { UiKit } from "../../../_infra/storyGroups";
```

Comment:
There are also some single quotes and unnecessary curly brackets around strings in this file.